openfisca_us/variables/gov/irs/credits/cdcc/count_cdcc_eligible.py

```diff
@@ -7,3 +7,8 @@ class count_cdcc_eligible(Variable):
     label = "CDCC-eligible children"
     unit = USD
     definition_period = YEAR
+
+    def formula(tax_unit, period, parameters):
+        num_eligible = add(tax_unit, period, ["cdcc_eligible"])
+        max_eligible = parameters(period).irs.credits.cdcc.eligibility.max
+        return min_(num_eligible, max_eligible)
```

Comment:
The law reads more as if the capping is done at the dollar amount stage, rather than there being a cap on the number of individuals:
>$6,000 if there are 2 or more [qualifying individuals](https://www.law.cornell.edu/definitions/uscode.php?width=840&height=800&iframe=true&def_id=26-USC-2002291164-1473529874&term_occur=999&term_src=title:26:subtitle:A:chapter:1:subchapter:A:part:IV:subpart:A:section:21) with respect to the taxpayer for such[ taxable year.](https://www.law.cornell.edu/definitions/uscode.php?width=840&height=800&iframe=true&def_id=26-USC-1529535480-1473532762&term_occur=999&term_src=)

So it might be closer to the law to move this capping into `cdcc_relevant_expenses`.